### PR TITLE
Update redirects-list.json

### PR DESCRIPTION
**Pull request by aguschin**
close https://github.com/iterative/mlem/issues/517#issuecomment-1340755113

### CODE REVIEW

**Comment by aguschin** [comment]
@iterative/websites, could you please review this? Don't know the syntax of this file.

I was trying to redirect `https://mlem.ai/doc/get-started/anypage` to `https://mlem.ai/doc/get-started`

**Comment by aguschin** [comment]
I run that locally and see that redirects doesn't work (`https://mlem.ai/doc/get-started/anypage` redirects to `https://mlem.ai/doc/get-startedanypage` for some reason). Can you maybe suggest a fix? Thanks!

**Comment by aguschin** [comment]
Ok, I debugged locally and fixed. Can be approved and merged now.

File: redirects-list.json
Comment (aguschin) [reply redirects-list.json:8]: yep, I used that file as a source of examples. What's the difference though?

Comment (jorgeorpinel) [reply redirects-list.json:8]: > What's the difference

In this case I think that there's no difference between `/(.+)` and `(/.*)?` since matches are not used later (`$1`, etc.) but not sure whether the `$` could make a difference... I like to add it to be more strict as a best practice. In general, `(/.*)?$` is tested and proven, but this is minor. Up to you/ @iterative/websites

Comment (jorgeorpinel) [reply redirects-list.json:8]: > making /doc/get-started redirect to /doc/get-started in an infinite loop

That's not happening in dvc.org atm but I do see your point. Should we also update the dvc.org and cml.dev repos with the simpler `/(.+)` ? Should we add `$` at the end too? Thanks

Comment (jorgeorpinel) [reply redirects-list.json:8]: > if we decided that the PR is ok

Sorry, not trying to block this @aguschin . It's pending a review by @iterative/websites I think. My only pending Q is whether the regex needs `$` at the end, really. I'll approve it for now.

**Review by jorgeorpinel — APPROVED** [review]

File: redirects-list.json
Comment (rogermparent) [reply redirects-list.json:8]: I don't think the $ at the end is required since the default match behavior is greedy, though I think the regex would also work with it and technically be more specific. Just a preference thing.

**Review by rogermparent — APPROVED** [review]
This looks fine to me, I only didn't review because I figured it'd be a content decision. Sorry if that blocked this!

**Comment by rogermparent** [comment]
I'll try a deploy to confirm for sure before merging!

File: redirects-list.json
Comment (jorgeorpinel) [reply redirects-list.json:8]: OK let's add it for consistency and do it's extra strict, I say.

```suggestion
  "^/doc/get-started/(.+)$                    /doc/get-started 302"
```

**Comment by rogermparent** [comment]
of course, we've got some residual gunk from the GH key issue. I think a `main` merge will make things work and we can try it out for real.

**Comment by rogermparent** [comment]
It works! 

https://user-images.githubusercontent.com/9111807/207182596-4c66d170-04c7-4c3a-bf7e-8d3e97f66d2f.mp4

File: redirects-list.json
Comment (rogermparent) [reply redirects-list.json]: I'll try it out for consistency's sake and see how it goes on the deploy.

**Comment by rogermparent** [comment]
Works the same way for me with @jorgeorpinel's `$` suggestion, looks like we're good to merge!

**Comment by jorgeorpinel** [comment]
Thanks @rogermparent

**Pull request merged**
